src/adagrams.js

```diff
@@ -1,7 +1,103 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+// hash to store frequencies of letters (values) and weights (keys)
+// probably could have just made one big array, but this felt more readable? 
+const LETTERFREQ = {1: ['J', 'K', 'Q', 'X', 'Z'], 
```

Comment:
Style: In JS we generally use `_`s to separate words in global constants like this.

```suggestion
const LETTER_FREQ = {1: ['J', 'K', 'Q', 'X', 'Z'], 
```